tests/e2e/specs/merchant/merchant-disputes-submit-winning.spec.js

```diff
@@ -131,6 +139,12 @@ describe( 'Disputes > Submit winning dispute', () => {
 				text: 'Evidence submitted!',
 			}
 		);
+
+		// Verify Won status in disputes view
+		await page.waitForSelector( 'span.chip-light' );
+		await expect( page ).toMatchElement( 'span.chip-light', {
```

Comment:
What do you think of specifically checking the top row of the table, or in some other way trying to establish that this text match pertains to the proper dispute?

Reply:
The problem is that sometimes Won status is not on the top, isn't it? But I have some flakiness now with existing code so I'll try to improve it.

Reply:
👍 Fine with me if we're thinking of it in terms of the broader test suite